src/window.rs

```diff
@@ -1017,37 +1017,44 @@ impl Window {
         self.window.set_window_icon(window_icon)
     }
 
-    /// Sets location of IME candidate box in client area coordinates relative to the top left.
+    /// Set the IME cursor editing area, where the `position` is the top left corner of that area
+    /// and `size` is the size of this area starting from the position.
     ///
-    /// This is the window / popup / overlay that allows you to select the desired characters.
-    /// The look of this box may differ between input devices, even on the same platform.
+    /// The windowing system could place a candidate box close to that area, but try to not obscure
+    /// the specified area.
```

Comment:
What I'm reading here is that `size` specifies an area, that the candidate box will **avoid**.
I assume what's meant here is that the candidate box should try to **stay inside** the specified area?

Reply:
No, it'll try to not obscure this area, simply so you can see what you're editing.

Reply:
Ah, I see!

> Set the IME cursor editing area, where the `position` is the top left corner of that area and `size` is the size of this area starting from the position.

That line was confusing to me then, because it's not defined what the IME cursor editing area is, sounds to me like it's the candidate box.

Reply:
I've added some clarification to that.